tests/logging_test.c

```diff
@@ -31,8 +31,6 @@
 #include "logging.h"
 #include "oci.h"
 
-gboolean enable_debug = true;
```

Comment:
Please can you also remove the `enable_debug` variables from the other `tests/*_test.c` files as they are now redundant.